client/src/components/ToolResults.tsx

```diff
@@ -156,7 +156,7 @@ const ToolResults = ({
         )}
         {structuredResult._meta && (
           <div className="mb-4">
-            <h5 className="font-semibold mb-2 text-sm">Meta:</h5>
+            <h5 className="font-semibold mb-2 text-sm">Meta Schema:</h5>
```

Comment:
This isn't a thing as far as I can tell. The spec only says

> The _meta property/parameter is reserved by MCP to allow clients and servers to attach additional metadata to their interactions.

If there is consensus or examples somewhere that you can point to of the tool `_meta` being interpreted as a schema, please note. 

Otherwise, there is the `inputSchema` that tools use to specify what they'd like the client to send in when invoking it.

Reply:
Same as above, I think we should stick with the more open Meta rather than indicate an specific intended use.